imagersite/imager_images/tests.py

```diff
@@ -128,3 +133,253 @@ def test_album_deletion_on_user_deletion(self):
         self.user1.delete()
         self.assertEqual(Album.objects.count(), 0)
 
+
+@override_settings(DEBUG=True)
+class LiveServerTest(StaticLiveServerTestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(LiveServerTest, cls).setUpClass()
+        cls.browser = Browser()
```

Comment:
That's true. I was trying to avoid having to restart a splinter browser session with each individual test (it's slow), but your comment makes sense I the sequential tests are testing discrete states.